### ISSUE TITLE
Weather binding: exception at startup in OH2

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab#4562]
The exception is not systematic.

```
03:36:21.246 [ERROR] [org.apache.felix.configadmin        ] - [org.osgi.service.cm.ManagedService, org.osgi.service.event.EventHandler, id=310, bundle=197/mvn:org.openhab.binding/org.openhab.binding.weather/1.9.0-SNAPSHOT]: Unexpected problem updating configuration org.openhab.weather
java.lang.IllegalStateException: Task already scheduled or cancelled
        at java.util.Timer.sched(Timer.java:401)[:1.8.0]
        at java.util.Timer.schedule(Timer.java:193)[:1.8.0]
        at org.openhab.binding.weather.internal.utils.DelayedExecutor.schedule(DelayedExecutor.java:44)[197:org.openhab.binding.weather:1.9.0.201607280110]
        at org.openhab.binding.weather.internal.scheduler.WeatherJobScheduler.restart(WeatherJobScheduler.java:65)[197:org.openhab.binding.weather:1.9.0.201607280110]
        at org.openhab.binding.weather.internal.bus.WeatherBinding.updated(WeatherBinding.java:86)[197:org.openhab.binding.weather:1.9.0.201607280110]
        at org.apache.felix.cm.impl.helper.ManagedServiceTracker.updated(ManagedServiceTracker.java:189)[7:org.apache.felix.configadmin:1.8.8]
        at org.apache.felix.cm.impl.helper.ManagedServiceTracker.updateService(ManagedServiceTracker.java:152)[7:org.apache.felix.configadmin:1.8.8]
        at org.apache.felix.cm.impl.helper.ManagedServiceTracker.provideConfiguration(ManagedServiceTracker.java:85)[7:org.apache.felix.configadmin:1.8.8]
        at org.apache.felix.cm.impl.ConfigurationManager$ManagedServiceUpdate.provide(ConfigurationManager.java:1444)[7:org.apache.felix.configadmin:1.8.8]
        at org.apache.felix.cm.impl.ConfigurationManager$ManagedServiceUpdate.run(ConfigurationManager.java:1400)[7:org.apache.felix.configadmin:1.8.8]
        at org.apache.felix.cm.impl.UpdateThread.run0(UpdateThread.java:143)[7:org.apache.felix.configadmin:1.8.8]
        at org.apache.felix.cm.impl.UpdateThread.run(UpdateThread.java:110)[7:org.apache.felix.configadmin:1.8.8]
        at java.lang.Thread.run(Thread.java:744)[:1.8.0]
```
##

**Comment by lolodomo** [comment]
This issue is still relevant.
No chance to fix it ?

**Comment by watou** [comment]
@gerrieg Would you be able to have a look?

**Comment by gerrieg** [comment]
should be fixed with https://github.com/openhab/openhab/pull/4896

**Comment by 9037568** [comment]
Closed via #4896.
